includedBuild/dependencies/src/main/kotlin/com/badoo/reaktive/dependencies/Deps.kt

```diff
@@ -2,17 +2,15 @@ package com.badoo.reaktive.dependencies
 
 object Deps {
 
-    private const val kotlinVersion = "1.5.30"
+    private const val kotlinVersion = "1.6.0"
     private const val coroutinesVersion = "1.5.2"
```

Comment:
Do we need to wait for coroutines?

Reply:
I heard they are compatible. Should be fine.